core/templates/dev/head/components/AnswerGroupEditorDirective.js

```diff
@@ -23,11 +23,14 @@ oppia.directive('answerGroupEditor', [
       scope: {
         isEditable: '=',
         displayFeedback: '=',
+        defaultAnswerGroup: '=',
         getOnSaveAnswerGroupDestFn: '&onSaveAnswerGroupDest',
         getOnSaveAnswerGroupFeedbackFn: '&onSaveAnswerGroupFeedback',
         getOnSaveAnswerGroupRulesFn: '&onSaveAnswerGroupRules',
+        getOnSaveAnswerGroupCorrectFn: '&onSaveAnswerGroupCorrect',
```

Comment:
getOnSaveAnswerGroupCorrectnessLabelFn: &onSaveAnswerGroupCorrectnessLabel

(A bit long, but you save a label, not a "correct".)

Reply:
Done!